Sources/Context.swift

```diff
@@ -8,11 +8,23 @@
 
 import Foundation
 
-public func enrich(context: [AnyHashable: Any], parameters: [String]) throws -> [AnyHashable: Any] {
+public enum StencilContext {
+  static let environment = "env"
+  static let parameters = "param"
+}
+
+/**
+ Enriches a stencil context with parsed parameters and environment variables
+ 
+ - Parameter context: The stencil context
+ - Parameter parameters: List of strings, will be parsed using the `Parameters.parse(items:)` method
+ - Parameter environment: Environment variables, defaults to `ProcessInfo().environment`
+ */
+public func enrich(context: [String: Any], parameters: [String], environment: [String: String] = ProcessInfo().environment) throws -> [String: Any] {
```

Comment:
Now that we have an enum, and because I'm not a fan of top-level functions, maybe it would be nice to move the `enrich` func as a `static func` inside `enum StencilContext`?